commands/root.go

```diff
@@ -0,0 +1,87 @@
+package commands
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+	"os/exec"
+	"os"
+	"github.com/mitchellh/go-homedir"
+	"log"
+	"github.com/spf13/viper"
+)
+
+const CONFIG_FILE_NAME = "settings.yaml"
+const CYBERNODE_SETTINGS_FOLDER = "/.cybernode/"
+
+var RootCmd = &cobra.Command{ Use: "cybernode" }
+
+func init() {
+	cobra.OnInitialize(findDocker, createCybernodeDirectory, initSettings)
+	RootCmd.AddCommand(SettingsCmd, ChainsCmd)
+}
+
+func findDocker() {
+	_, err := exec.LookPath("docker")
+	if err != nil {
+		log.Fatal("You should install docker in order to use cybernode. Please visit https://docs.docker.com/install/ for more info")
+	}
+}
+
+func initSettings() {
+
+	settingsFilePath := getSettingsFilePath()
+
+	if _, err := os.Stat(settingsFilePath); err != nil {
+
+		if os.IsNotExist(err) {
```

Comment:
err? or here should be path?

Reply:
This method checks if error is of type NotExist